### ISSUE TITLE
./jive: symbol lookup error: /lib64/libsndfile.so.1: undefined symbol: ogg_stream_flush_fill

### ISSUE THREAD

**Issue by xpseudonym** [ralph-irving/squeezeplay#24]
Hi Ralph,
After install as per the instructions here [README.txt](https://sourceforge.net/projects/lmsclients/files/squeezeplay/linux/README.txt/download)
On go I get:
```
$ ./squeezeplay.sh
./jive: symbol lookup error: /lib64/libsndfile.so.1: undefined symbol: ogg_stream_flush_fill
```
Any ideas?
Many thanks
Morgan

**Comment by xpseudonym** [comment]
This seems fixed by install of `squeezeplay-8.4.1-1485-x86_64.tgz` instead of `squeezeplay-pulse-8.4.1-1485-x86_64.tgz`: closing.  Thx

**Comment by Lopasilver** [comment]
Thank you.
Got same error message, no-pulse version is working fine
